cmd/open.js

```diff
@@ -10,13 +10,43 @@ exports.builder = (yargs) =>
     yargs
     .strict()
     .usage('Usage: $0 open <package> [options]\n\nOpens the package in your browser..')
-    .example('$0 open gulp', 'Opens "gulp" package')
-    .demand(1, 1);
+    .example('$0 open gulp', 'Opens "gulp" package using auto source')
+    .example('$0 open --link npms gulp', 'Opens "gulp" package in `https://npms.io` service')
```

Comment:
Minor change here, can this be `$0 open gulp --link npms`? I think it's less confusing.